include/cutlass/gemm/device/gemm.h

```diff
@@ -495,7 +495,18 @@ class Gemm {
     const auto sycl_block = syclcompat::dim3(block.x, block.y, block.z);
     const auto sycl_grid = syclcompat::dim3(grid.x, grid.y, grid.z);
 
-    syclcompat::launch<cutlass::Kernel<GemmKernel>>(sycl_grid, sycl_block, smem_size, params_);
+    auto q = stream ? *stream : syclcompat::get_default_queue();
```

Comment:
Why does this make a copy of the queue instead of `auto& q = ...`? I don't expect much overhead. But it appears unnecessary nonetheless.

Reply:
Sorry for the late reply. I that is just me overlooking that option.